src/main/webapp/usermap.html

```diff
@@ -55,8 +55,15 @@
             <h1 id="caption">Who is using TEAMMATES?</h1>
             <div id="contentHolder">
                 <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE"></div>
-                <p id="totalCount"></p>
-                <p>Last updated: 14 March 2016</p>
+                <p id="lastUpdate" class="lastUpdate">Last updated: 14 March 2016</p>
+                <h2 class="subcaption align-center">
+                <span id="totalUserCount" class="totalCount">
+                </span> 
+                institutions from 
+                <span id="totalCountryCount" class="totalCount">
+                </span> 
+                countries
```

Comment:
Hmmm you misunderstood the comment. What I was asking was to indent the block from `<span>` to `countries` one level further because they're the child element of the H2 element. So it'll be something like:
```html
<h2>
    <span ...>
    ...
    countries
</h2>
```
The closing `</span>` is fine how it was before or now, although I'd prefer how it was before because we're saving one line break each that way.